flink-core/src/main/java/org/apache/flink/api/common/typeutils/TypeSerializerSerializationUtil.java

```diff
@@ -69,7 +69,7 @@
 	 * written using {@link #writeSerializer(DataOutputView, TypeSerializer)}.
 	 *
 	 * <p>If deserialization fails for any reason (corrupted serializer bytes, serializer class
-	 * no longer in classpath, serializer class no longer valid, etc.), {@code null} will
+	 * no longer in classpath, serializer class no longer valid, etc.), an {@link IOException} is thrown.
```

Comment:
Comment in the next line should be deleted, looks like leftover from the copy-paste.